group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -579,7 +579,7 @@ public List<DescribeGroupsResponseData.DescribedGroup> describeGroups(
     }
 
     /**
-     * Gets or maybe creates a consumer group.
+     * Gets or maybe creates a consumer group without updating the groups map.
```

Comment:
nit: Could we extend the comment to mention that the group will be materialized during the replay?